src/pages/Lend/LendByCategoryPage.vue

```diff
@@ -723,7 +723,8 @@ export default {
 			this.setLeftArrowPosition();
 		}
 
-		if (this.$route.query.utm_campaign === 'liked_loan' || this.$route.query.utm_campaign.includes('liked_loan')) {
+		if (this.$route.query.utm_campaign && (!!this.$route.query.utm_campaign === 'liked_loan'
```

Comment:
@roger-in-kiva I think this can be simplified to a single check using optional chaining. Something like:

`if (this.$route?.query?.utm_campaign?.includes('liked_loan')) { // code ...}`

Reply:
Oh yeah, and working fine!